### PR TITLE
examples: choose panda for fw query

### PR DESCRIPTION
**Pull request by incognitojam**
improves the car port experience for those needing multiple pandas/external red panda

https://discord.com/channels/469524606043160576/539096103468007424/1022224299093921913

### CODE REVIEW

**Comment by incognitojam** [comment]
maybe it should translate `--bus 4` and others to the correct panda?

File: python/__init__.py [hunk 1]
```diff
@@ -541,6 +541,27 @@ def get_type(self):
 
     return ret
 
+  def get_type_name(self):
+    hw_type = self.get_type()
+    if hw_type == Panda.HW_TYPE_WHITE_PANDA:
```
Comment (adeebshihadeh) [inline python/__init__.py:546]: use a dict

Comment (incognitojam) [reply python/__init__.py:546]: I tried but it said it was unhashable

Comment (incognitojam) [reply python/__init__.py:546]: ah I have an idea

**Comment by incognitojam** [comment]
better to improve fw_versions.py in openpilot instead  https://github.com/commaai/openpilot/blob/master/selfdrive/car/fw_versions.py

**Comment by sshane** [comment]
query_fw_versions.py is useful for debugging, would this fix red panda querying?

File: python/__init__.py [hunk 2]
```diff
@@ -183,6 +183,18 @@ class Panda:
   HW_TYPE_RED_PANDA_V2 = b'\x08'
   HW_TYPE_TRES = b'\x09'
 
+  HW_TYPES = [HW_TYPE_WHITE_PANDA, HW_TYPE_GREY_PANDA, HW_TYPE_BLACK_PANDA, HW_TYPE_PEDAL, HW_TYPE_UNO, HW_TYPE_DOS, HW_TYPE_RED_PANDA, HW_TYPE_RED_PANDA_V2]
```
Comment (sshane) [inline python/__init__.py:186]:
```suggestion
```

**Pull request merged**